### DISCUSSION TITLE
ROI settings not saved completly after reboot

### DISCUSSION THREAD

**Discussion by ThePhantom79** [jomjol/AI-on-the-edge-device#2146] [Q&A]
Version I use: 15.0.3

I disable the tick boxes "Lock spect ratio", "Synchronize ... " and "Keep ..", because I have two different sizes of numbers in one row. First digits before "," are bigger then after.
Unfortunately these tickboxes were not saved after Save and Reboot, and for that the exact setting for the location of the diggits change after reboot.
You can see the numbers for "size x" and "size y" changes after reboot!
Thats really bad for the recognition, as I really put the right settings in there with a lot of work, but after save and reboot it changes always ...

**Comment by caco3** [comment]
those checkboxes are only used in the webbrowser but not stored on the device.
it is right that the ROI pages mess your settings up again on a page reload (eg. because of the reboot). but as long as you do not save again, it is working correctly (beside the wrong visualization on the ROI pages).

**Reply by caco3** [reply]
maybe we can improve this behaviour in https://github.com/jomjol/AI-on-the-edge-device/issues/2148
